### PR TITLE
Docker Ready State

### PR DESCRIPTION
**Pull request by ravolt**
Add a basic docker roles that help with Docker machine provisioning. The "docker-ready" is a milestone that is used by Docker Machine.

The "docker-prep" role is used to install items that may be needed (but assumed) by Docker machine. This includes apparmour, cgroups-lite and lxc

For now, it only works on Ubuntu (just like Docker Machine)

### CODE REVIEW

**Comment by zehicle** [comment]
connect to #601

**Comment by galthaus** [comment]
Sigh - this needs a rebase after pulling in my changes.  Dang it!

**Comment by zehicle** [comment]
@galthaus you and I had both added to the end of the crowbar.yml.  Should be corrected to have BOTH additions.

File: crowbar.yml [hunk 1]
```diff
@@ -65,6 +65,8 @@ barclamps:
     description: "Consul Service Monitoring and Management"
   - name: sysdig
```
Comment (zehicle) [inline crowbar.yml]: this was the conflict item - resolved to include both

**Comment by sjensen** [comment]
ignore for now for testing

**Pull request merged**